packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2109,36 +2110,59 @@ export class ProjectService {
         if (!explores) {
             return [];
         }
-        const allExploreSummaries = explores.map<SummaryExplore>((explore) => {
-            if (isExploreError(explore)) {
-                return {
-                    name: explore.name,
-                    label: explore.label,
-                    tags: explore.tags,
-                    groupLabel: explore.groupLabel,
-                    errors: explore.errors,
-                    databaseName:
-                        explore.baseTable &&
-                        explore.tables?.[explore.baseTable]?.database,
-                    schemaName:
-                        explore.baseTable &&
-                        explore.tables?.[explore.baseTable]?.schema,
-                    description:
-                        explore.baseTable &&
-                        explore.tables?.[explore.baseTable]?.description,
-                };
-            }
+        const userAttributes =
+            await this.userAttributesModel.getAttributeValuesForOrgMember({
+                organizationUuid,
+                userUuid: user.userUuid,
+            });
 
-            return {
-                name: explore.name,
-                label: explore.label,
-                tags: explore.tags,
-                groupLabel: explore.groupLabel,
-                databaseName: explore.tables[explore.baseTable].database,
-                schemaName: explore.tables[explore.baseTable].schema,
-                description: explore.tables[explore.baseTable].description,
-            };
-        });
+        const allExploreSummaries = explores.reduce<SummaryExplore[]>(
```

Comment:
convert map into a reduce so we can also filter out the explores the user doesn't have access to in the same loop